pkg/console/install_panels.go

```diff
@@ -1188,6 +1192,13 @@ func addHostnamePanel(c *Console) error {
 		return err
 	}
 
+	askDhclientSetHostnameV, err := widgets.NewDropDown(c.Gui, askDhclientSetHostnamePanel, "Set via DHCP", func() ([]widgets.Option, error) {
+		return []widgets.Option{{Value: "no", Text: "No"}, {Value: "yes", Text: "Yes"}}, nil
```

Comment:
I'll also suggest adding a warning message if the user selects `yes` to get the hostname from the DHCP server. The idea is that the hostname should be **always** assigned persistently by the DHCP server. hostname changes can lead to unexpected consequences. etcd, certificates, and many services rely on the hostname.

Reply:
Yep. If we do take this change (rather than the alternative in #726) adding a warning here seems like a very good idea.